modules/authentication/contexts/authentication-context.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const AuthenticationContext = React.createContext(undefined);
```

Comment:
I have the same doubt. If is it better to explicitly import the createContext method.

Reply:
I think that for convention it's done the way it is: https://reactjs.org/docs/context.html and also check @benjakramm example: https://github.com/btkramm/doble-clic/blob/main/src/modules/authentication/contexts/authentication-context.ts

Reply:
I understand. You are right then.

Reply:
donee